rfc-010-core-skill-examinations.md

```diff
@@ -0,0 +1,25 @@
+# Core Skill Examinations
+
+## Summary
+
+Decide who is justified to determine somebody's achievement of a core skill badge.
+
+## Problem
+
+We recently started core skill tests with a TDD curriculum of three badges. The curriculum was primarily developed by Adrian, Craig and Richard, with contributions by other developers and directors. After Adrian, Craig and Richard began invigilating these tests, there was concern amongst the team over the rules of who may invigilate.
+
+- Who should be able to invigilate a test?
+- What criteria is necessary for somebody to have this privilege?
+- How can we ensure that assessment is fairly given?
+- If we involve invigilators who did not write curriculum, how can we ensure they understand the marking scheme enough to fairly assess others?
+- What are the rules???
+
+## Proposal
+
+1. Marking schemes and curriculum designed **MUST** be publicly distributed amongst all team members who may want to sit the exam before examinations start. Any team member **MAY** question or criticise the material, and their feedback **MUST** be considered before starting examinations. Time should be given for this feedback. This prevents course leaders from assessing prematurely, if the team feels the material needs work.
```

Comment:
How does feedback apply retroactively?

Reply:
Thought about that - I think it may be out of scope of this, but my view is if the material is consented to by the team at time of writing, it should remain suitable even after new members join?